lymph/unilateral.py

```diff
@@ -17,7 +17,8 @@
 from lymph.edge import Edge
 from lymph.node import LymphNodeLevel, Tumor
 from lymph.timemarg import MarginalizorDict
-from tests.unit.custom_strategies import nodes
+
+# from tests.unit.custom_strategies import nodes

```

Comment:
Just delete it, if you don't need it. The nice thing about version control with git is that you don't need to worry about loosing stuff.

Reply:
I got errors that I could not fix, so I commented it. I wanted to ask you what's the meaning this import.

Reply:
I'm not sure actually, it's an unused import and doesn't make sense here 😅

Reply:
Hahaha, good, it's gone 👍🏽